src/parsers.rs

```diff
@@ -17,6 +21,15 @@ impl<'a> Parser<'a> {
         }
     }
 
+    #[cfg(feature = "ipv6")]
```

Comment:
Let's mark this entire file `#[cfg_attr(not(and(feature = "proto-ipv4", feature = "proto-ipv6")), allow(dead_code)]`.

Reply:
Added `#[cfg_attr(not(feature = "proto-ipv6"), allow(dead_code))]`. Will add the `proto-ipv4` case once I move on to the `proto-ipv4` feature component of #76